shared/src/store/action-reducers/simulation.ts

```diff
@@ -1085,13 +1309,39 @@ export namespace SimulationActionReducers {
         {
             action: ProvidePersonnelBehaviorUpdateVehiclePrioritiesAction,
             reducer(draftState, { simulatedRegionId, behaviorId, priorities }) {
+                const simulatedRegion = getElement(
+                    draftState,
+                    'simulatedRegion',
+                    simulatedRegionId
+                );
                 const behaviorState = getBehaviorById(
                     draftState,
                     simulatedRegionId,
                     behaviorId,
                     'providePersonnelBehavior'
                 );
 
+                let prioritiesString = '';
+                let i = 1;
+                priorities.forEach((priority) => {
+                    prioritiesString += `${i}.`;
+                    prioritiesString += draftState.vehicleTemplates.find(
+                        (vehicleTemplate) => vehicleTemplate.id === priority
+                    )!.name;
+                    prioritiesString += ' ';
+                    i++;
+                });
```

Comment:
```suggestion
                const prioritiesString = priorities.map((priority, i) => {
                    const name = draftState.vehicleTemplates.find(
                        (vehicleTemplate) => vehicleTemplate.id === priority
                    )!.name;
                    return `${i}. ${name}`
                }).join(' ');
```
Haven't tested it, but this looks better imo.

You can also remove the `trimEnd` then.

Reply:
d